pyfar/classes/coordinates.py

```diff
@@ -1357,8 +1357,8 @@ def show(self, mask=None, **kwargs):
         Parameters
         ----------
         mask : boolean numpy array, None, optional
-            Plot points in red if ``mask==True``. The default is ``None``,
-            which the same color for all points.
+            Mask or indexes to plot. Plot points in red if ``mask==True``.
+            The default is ``None``, which the same color for all points.
```

Comment:
```suggestion
            Mask or indexes to highlight. Highlight points in red if ``mask==True``.
            The default is ``None``, which plots all points in the same color.
```
Also, is indexes correct here? For me that would be an array of integers. In turn `mask==True` would not make sense. But I might be overthinking it.